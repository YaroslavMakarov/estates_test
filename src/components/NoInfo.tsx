import React from 'react';
import { Container } from 'react-bootstrap';
import 'bootstrap/dist/css/bootstrap.min.css';

import { NoInfoContent } from '../styled/NoInfoStyles';

const NoInfo = () => (
        <Container className="d-flex align-items-center justify-content-center">
            <NoInfoContent>
                Sorry, don't have any information about this estate.
            </NoInfoContent>
        </Container>
);

export default NoInfo;