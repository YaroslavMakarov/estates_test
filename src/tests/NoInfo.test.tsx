import React from 'react';
import { render } from '@testing-library/react';

import NoInfo from '../components/NoInfo';

describe('BackButton component', () => {
    const wrapper = render(<NoInfo />);

    it('get NoInfo snapshot', () => {
        expect(wrapper.baseElement).toMatchSnapshot();
    });
});