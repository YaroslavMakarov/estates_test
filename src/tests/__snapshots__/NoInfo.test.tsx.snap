// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackButton component get BackButton snapshot 1`] = `
<body>
  <div>
    <div
      class="d-flex align-items-center justify-content-center container"
    >
      <div
        class="sc-bdfBwQ fciMqB"
      >
        Sorry, don't have any information about this estate.
      </div>
    </div>
  </div>
</body>
`;
